### DISCUSSION TITLE
SEB failed to start a new session

### DISCUSSION THREAD

**Discussion by Di10069** [SafeExamBrowser/seb-win-refactoring#608] [General]
[2023-05-08_17h10m51s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/11421882/2023-05-08_17h10m51s_Runtime.log)
![Screenshot_1](https://user-images.githubusercontent.com/132918444/236847749-4caac139-6c0d-4048-9ee2-4e8b95a4547e.png)
Whenever I tried to open it, it shows SEB failed to start a new session. Please help me with this

**Comment by dbuechel** [comment]
You're using the SEB Service, and it appears it cannot successfully perform the lockdown for a new session:

`2023-05-08 17:11:29.430 [04] - ERROR: Failed to start new service session within 30 seconds!`

To know why, we'd need the service log file as well, as there the exact reason of failure should be stated.

**Reply by Di10069** [reply]
[2023-05-08_14h01m02s_Service.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/11422601/2023-05-08_14h01m02s_Service.log)
 I don't have the service log file  17h10m. That's the last one.

**Reply by dbuechel** [reply]
Hm, that is weird indeed. Either way, you're using an outdated version of SEB, please try to update to the latest release version 3.5.0 and try again.

**Reply by Di10069** [reply]
[2023-05-10_18h04m17s_Service.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/11443466/2023-05-10_18h04m17s_Service.log)
[2023-05-10_18h04m17s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/11443467/2023-05-10_18h04m17s_Runtime.log)

still not working...

**Reply by dbuechel** [reply]
Okay, now there appears to be another problem:

```
2023-05-10 18:04:30.555 [06] - ERROR: [EaseOfAccessConfiguration] Failed to set registry item 'HKEY_LOCAL_MACHINE\Software\Microsoft\Windows NT\CurrentVersion\Image File Execution Options\Utilman.exe\Debugger' => 'SebDummy.exe'!

   Exception Message: Попытка выполнить несанкционированную операцию.
   Exception Type: System.UnauthorizedAccessException

   в Microsoft.Win32.RegistryKey.Win32Error(Int32 errorCode, String str)
   в Microsoft.Win32.RegistryKey.SetValue(String name, Object value, RegistryValueKind valueKind)
   в Microsoft.Win32.Registry.SetValue(String keyName, String valueName, Object value, RegistryValueKind valueKind)
   в SafeExamBrowser.Lockdown.FeatureConfigurations.RegistryConfigurations.RegistryConfiguration.TrySet(RegistryDataItem item) в C:\Users\appveyor\projects\seb-win-refactoring-ist041kp6jai2xxx\SafeExamBrowser.Lockdown\FeatureConfigurations\RegistryConfigurations\RegistryConfiguration.cs:строка 229

2023-05-10 18:04:30.555 [06] - WARNING: [EaseOfAccessConfiguration] Failed to disable feature!
2023-05-10 18:04:30.555 [06] - ERROR: Failed to configure EaseOfAccessConfiguration (f973a033-a4a2-40bf-bad1-82fef06fc87a)!
2023-05-10 18:04:30.555 [06] - ERROR: Lockdown was not successful!
```

So you'd need to check whether there is something interfering with the lockdown functionality of SEB, specifically something like an anti-virus or security software which interferes when SEB attempts to make changes to the Windows registry.

**Reply by dbuechel** [reply]
The application, system or security event log of Windows itself might give any hints, to be found in the Event Viewer system utility.

**Reply by Di10069** [reply]
Hello again, I can't start SEB. Background tasks are closed, antivirus uninstalled, SEB reinstalled. However, when I try to start the SEB again gives errors. Screenshots and logs are attached. 

[2023-05-13_01h12m48s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/11468200/2023-05-13_01h12m48s_Runtime.log)
![8990](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/132918444/8e3b36ca-80f0-4d9e-811e-63d6a8e44394)

![656](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/132918444/34993e5a-702e-49f2-89c9-98df75e70a62)

**Reply by dbuechel** [reply]
As the log already shows, the resource appears to not be available (be it due to network issues or to a problem on your computer system):

```
2023-05-13 01:13:31.514 [06] - ERROR: [NetworkResourceLoader] Failed to check availability of 'sebs://allenamento.cisiaonline.it/seb/student_cfg/seb_config_file_...' via GET request!

   Exception Message: Произошла ошибка при отправке запроса.
   Exception Type: System.Net.Http.HttpRequestException

   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в SafeExamBrowser.Configuration.DataResources.NetworkResourceLoader.<>c__DisplayClass11_0.<<Execute>b__0>d.MoveNext() в C:\Users\appveyor\projects\seb-win-refactoring-ist041kp6jai2xxx\SafeExamBrowser.Configuration\DataResources\NetworkResourceLoader.cs:строка 116
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в SafeExamBrowser.Configuration.DataResources.NetworkResourceLoader.Execute(HttpRequestMessage request) в C:\Users\appveyor\projects\seb-win-refactoring-ist041kp6jai2xxx\SafeExamBrowser.Configuration\DataResources\NetworkResourceLoader.cs:строка 120
   в SafeExamBrowser.Configuration.DataResources.NetworkResourceLoader.IsAvailable(Uri resource) в C:\Users\appveyor\projects\seb-win-refactoring-ist041kp6jai2xxx\SafeExamBrowser.Configuration\DataResources\NetworkResourceLoader.cs:строка 189

   Inner Exception Message: Невозможно соединиться с удаленным сервером
   Inner Exception Type: System.Net.WebException

   в System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   в System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)

   Inner Exception Message: Попытка установить соединение была безуспешной, т.к. от другого компьютера за требуемое время не получен нужный отклик, или было разорвано уже установленное соединение из-за неверного отклика уже подключенного компьютера 90.147.201.43:443
   Inner Exception Type: System.Net.Sockets.SocketException

   в System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   в System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   в System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)

2023-05-13 01:13:31.518 [06] - DEBUG: [NetworkResourceLoader] Can't load 'sebs://allenamento.cisiaonline.it/seb/student_cfg/seb_config_file_...' since its URI scheme is not supported or the resource is unavailable.
```

**Comment by MarcDietrich21** [comment]
Hello
I hope, I am here on the right place with my question...

I have the same problem. SEB worked. I tried some school-tests. Therefore I locked in and out in IsTest2 (students sight). Suddendly it did not work anymore. 
Above I can not see a solutions for this problem. The error massages are the same than shown 
Thank you for helping!


[2023-06-07_21h45m41s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/11712477/2023-06-07_21h45m41s_Runtime.log)
![SEB1](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/100416176/33c30e93-888f-47e8-ace0-93e015833f4f)
![SEB2](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/100416176/667e248d-ce6d-4272-889c-bbb5d308291f)
![SEB3](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/100416176/c9c732ee-83a2-48b4-9aa9-d0621371ef3f)

**Reply by dbuechel** [reply]
@MarcDietrich21 In your case, the configuration file contains invalid data (as is stated in the runtime log file):

```
2023-06-07 21:45:42.392 [04] - ERROR: [ConfigurationRepository] Unexpected error while trying to load 'sebs://www.istest2.ch//sebfiles/[...].seb'!

   Exception Message: Unerwartetes Token '='. Erwartet wurde das Token ';'. Zeile 31, Position 80.
   Exception Type: System.Xml.XmlException

   [...]

2023-06-07 21:46:11.718 [04] - WARNING: The configuration has failed!
```

So you'd need to get in contact with the manufacturer of your LMS in order to sort the issue out. Please feel free to post that particular part (line 31, column 80) of the configuration file here in order to determine exactly what is wrong in the configuration data itself.

**Comment by dbuechel** [comment]
As a general comment: "Session Start Error" / "SEB failed to start a new session" is a generic error always displayed in case SEB fails to start a new session or fails to startup correctly. That does not mean that every issue is the same, thus please always open a new discussion / issue unless the exact reason of failure (which normally is visible in the first shown error message or then in the application log, as indicated in the session start error message) is the same as the one specified in an already existing discussion or issue.

**Comment by KollatiKamakshi** [comment]
Same Issue...Please anyone solve this issue.. 
[2025-05-26_08h41m06s_Runtime.log](https://github.com/user-attachments/files/20434439/2025-05-26_08h41m06s_Runtime.log)
[2025-05-26_08h41m06s_Service.log](https://github.com/user-attachments/files/20434440/2025-05-26_08h41m06s_Service.log)

[]
![SEBStartupfailure](https://github.com/user-attachments/assets/63ccb16d-5603-44e9-a4a0-32080bb1a225)
(url)

**Reply by dbuechel** [reply]
We don't provide support for custom builds.

**Reply by KollatiKamakshi** [reply]
But Why..?

**Reply by SAITEJA784255** [reply]
![Uploading స్క్రీన్‌షాట్ 2025-07-07 230708.png…]()

**Comment by dbuechel** [comment]
This discussion has become too broad resp. concerning many (often unrelated) topics as the error message of the OP above is generic and may have many different causes.
